common/lib/capa/capa/tests/test_input_templates.py

```diff
@@ -502,6 +545,19 @@ def test_size(self):
 
         self.assert_has_xpath(xml, "//input[@size='40']", self.context)
 
+    def test_description(self):
+        """
+        Test that correct description information is set on desired elements.
+        """
+        xml = self.render_to_xml(self.context)
+
+        # Verify that input aria-describedby attribute has correct description ids
+        fieldset_aria_describedby = xml.xpath("string(//input/@aria-describedby)")
```

Comment:
Create a helper method that these tests can call? It looks like the only difference is the CSS selector.